.azure-pipelines/win32-conda-CI.yml

```diff
@@ -68,6 +68,7 @@ jobs:
       echo Test numpy installation... && python -c "import numpy"
       python -m pip install %COREML_PATH% %ONNX_PATH%
       python -m pip install tensorflow-cpu==1.15.0
+      python -m pip install torch==1.5.0+cpu -f https://download.pytorch.org/whl/torch_stable.html
```

Comment:
Agreed! Pytorch dependency is actually in Hummingbird requirements, however [we found](https://github.com/microsoft/hummingbird/issues/166#issue-644317532) that pytorch is not update on pip for windows. Another option is to install hummingbird directly from source rather than from pip. That would work. Should I install Hummingbird from github?

Reply:
In that case, can we put this line in the hummingbird requirements.txt : torch==1.5.0+cpu -f https://download.pytorch.org/whl/torch_stable.html, 
Or add some lines in the code to log this issue or avoid the confusion?

Reply:
This is complicated because `torch==1.5.0+cpu -f https://download.pytorch.org/whl/torch_stable.html` will not work for linux and mac. We have [this](https://github.com/microsoft/hummingbird/blob/40d8cc9477fc598081b973217e1315c9a0a890c7/setup.py#L25) logic to install PyTorch, but [we are having problems with pip from windows](https://github.com/microsoft/hummingbird/issues/166). I think that installing directly from github is probably the best solution right now.

Reply:
Ok installing from github does not work for python 3.7 because of [this](https://dev.azure.com/onnxmltools/onnxmltools/_build/results?buildId=13929&view=logs&j=14d29e63-7f60-55a7-fdd9-83f0bcb9c249&t=d805671f-b036-5223-a743-bcd6fb46af56). Basically `onnx` uses `typing` which for some reason creates problem with Hummingbird. Since you are already manually installing tensorflor, is it a problem if for windows we also manually install pytorch in the pipeline?